jerry-core/jcontext/jcontext.h

```diff
@@ -187,6 +190,36 @@ extern jerry_hash_table_t jerry_global_hash_table;
 
 #endif /* !CONFIG_ECMA_LCACHE_DISABLE */
 
+#else /* JERRY_ENABLE_EXTERNAL_CONTEXT */
+
+#define JERRY_CONTEXT(field) (((jerry_context_t *) jerry_port_get_current_instance()->context_p)->field)
```

Comment:
I'm suggesting to changing this to:

```
#define JERRY_CONTEXT(field) (((jerry_context_t *)jerry_port_get_current_context())->field)
```

For every place `JERRY_CONTEXT` is used (~280 times, see below), this alternative would probably result in smaller binary code size, because there's no need for the `->context_p` dereferencing (so no load instruction or whatever it compiles to).

```
$ find ./jerry-core -iname *.c | xargs cat | grep -e JERRY_CONTEXT | wc -l
     282
```